pkg/virt-controller/services/template.go

```diff
@@ -1104,7 +1107,7 @@ func (t *templateService) RenderHotplugAttachmentTriggerPodTemplate(volume *v1.V
 func (t *templateService) RenderExporterManifest(vmExport *exportv1.VirtualMachineExport, namePrefix string) *k8sv1.Pod {
 	exporterPod := &k8sv1.Pod{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      fmt.Sprintf("%s-%s", namePrefix, vmExport.Name),
+			Name:      naming.GetName(namePrefix, vmExport.Name, validation.DNS1035LabelMaxLength),
```

Comment:
As @mhenriks pointed out in my PR, naming.GetPodName is slightly more appropriate here, no need to pass the validation parameter (same otherwise).

Reply:
Makes sense - updated.